Map/RoomBase.gd

```diff
@@ -4,12 +4,26 @@ class_name RoomBase
 ##
 ## Holds basic functionality for a room on the map
 
-@export var light_level: int = 0
+@export var light_level: Enums.LightLevel = Enums.LightLevel.UNLIT
```

Comment:
this should be done in an _init function, if you have classes that extend from `RoomBase` it seems like the _init() of those child class to change default value doesn't work